python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3451,7 +3451,6 @@ def test_can_create_multiple_sessions_to_different_remotes(self):
         # Gets currently active session.
         same = PySparkSession.builder.remote("sc://other.remote.host:114/").getOrCreate()
         self.assertEquals(other, same)
-        same.stop()
```

Comment:
@HyukjinKwon could you suggest how to help with it?
The issue is that this test uses a bunch of sessions with dummy hosts, but these sessions used to never issue any RPCs, but now they do ReleaseSession in the stop, and then it notices that the host doesn't work and it fails...
I cannot simply remove the stop, because these tests actually test how stop()
In scala I resolved it with a bit of hacks in https://github.com/apache/spark/pull/43546/files#diff-a2b06ed831053e086e47d191c1e86d26da81c2d2a5cbaedd34a712297a2b418dR126
https://github.com/apache/spark/pull/43546/files#diff-a2b06ed831053e086e47d191c1e86d26da81c2d2a5cbaedd34a712297a2b418dR126 ...

Reply:
@juliuszsompolski I will take a look soon. Feel free to skip the tests for now, and we can just merge this first if this is an issue in tests only